consensus/dummy/dynamic_fees.go

```diff
@@ -83,14 +85,14 @@ func CalcBaseFee(config *params.ChainConfig, parent *types.Header, timestamp uin
 	case isApricotPhase4:
 		// The blockGasCost is paid by the effective tips in the block using
 		// the block's value of baseFee.
-		blockGasCost = calcBlockGasCost(
-			ApricotPhase4TargetBlockRate,
-			ApricotPhase4MinBlockGasCost,
-			ApricotPhase4MaxBlockGasCost,
-			ApricotPhase4BlockGasCostStep,
+		//
+		// Although the child block may be in AP5 here, the blockGasCost is
+		// still calculated using the AP4 step.
```

Comment:
can we also explain why this is the case?

Reply:
I don't think there is a good reason... I had assumed it was a bug

Reply:
can we document it as such (preserving existing behavior?)